src/JustSaying/Fluent/QueueSubscriptionBuilder`1.cs

```diff
@@ -95,43 +147,77 @@ public QueueSubscriptionBuilder<T> WithName(string name)
         /// <summary>
         /// Configures the SQS read configuration.
         /// </summary>
-        /// <param name="configure">A delegate to a method to use to configure SQS reads.</param>
+        /// <param queueName="configure">A delegate to a method to use to configure SQS reads.</param>
         /// <returns>
         /// The current <see cref="QueueSubscriptionBuilder{T}"/>.
         /// </returns>
         /// <exception cref="ArgumentNullException">
-        /// <paramref name="configure"/> is <see langword="null"/>.
+        /// <paramref queueName="configure"/> is <see langword="null"/>.
         /// </exception>
         public QueueSubscriptionBuilder<T> WithReadConfiguration(Action<SqsReadConfiguration> configure)
         {
             ConfigureReads = configure ?? throw new ArgumentNullException(nameof(configure));
             return this;
         }
 
+        /// <summary>
+        /// Overrides the policy of generating the topic from the type queueName, and allows it to be passed in instead
+        /// </summary>
+        /// <param queueName="topicName">The topic queueName to use</param>
+        /// <param queueName="topicARN">We do not supply a queueName, instead use this externally created Arn instead. Should be used with validate infrastructure only</param>
+        /// <returns></returns>
+        /// <exception cref="InvalidOperationException"></exception>
+        public QueueSubscriptionBuilder<T> WithTopic(string topicName = null, string topicARN = null)
```

Comment:
```suggestion
        public QueueSubscriptionBuilder<T> WithTopic(string topicName = null, string topicArn = null)
```
We follow the .NET naming convention for 3 letter initialisms just like the AWS SDK does